src/components/Home/index.js

```diff
@@ -49,6 +51,26 @@ export default class Home extends React.Component {
     let totalPromises = 1;
     let numberFinished = 0;
 
+    let eventId = urlParamsHandler.getUrlParameter('eventId');
+    var stateId = urlParamsHandler.getUrlParameter('state');
```

Comment:
nit, and I need to update the eslint rules, but can you use `const` or `let` instead of `var`